### DISCUSSION TITLE
Multiple notification mails are being sent when using the same email address for multiple options in a dynamic recipients field

### DISCUSSION THREAD

**Discussion by FrDH** [solspace/craft-freeform#739] [Questions & Answers] [answered]
**Describe the bug or issue you're experiencing**
The admin notification mail is sent multiple times to the same address when that address is used for multiple options in a dynamic recipients field.

**Steps to reproduce**

1. Create a form with a dynamic recipiets field and populate the field with a few options that all have the same email address.
2. Add the form to a page, select one of the options from the field.
3. Submit the form and check for emails.

**Expected behavior**
I'd expect a single email notification to be sent.

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 3.8.6
- Freeform Version: 3.13.28
- Freeform Edition:PRO
- Fresh Install or Upgrade: Fresh install

**Additional context**

**Comment by kjmartens** [comment, marked as answer]
Hi @FrDH,

This is a [current limitation of Freeform](https://docs.solspace.com/craft/freeform/v4/common-issues/#multiple-emails-being-sent-for-dynamic-recipients-field), as Freeform will perceive the user submitting all of the matching options at once, sending off emails as many times as there are duplicate options.

There are three workarounds for this:

1. Create an email address alias for each option so that each option in the Dynamic Recipients field type settings will be unique.
2. Add sub-address tags to the affected email addresses so their values are unique. For example: `joe+whatever@example.com`, `joe+somethingelse@example.com`, etc
3. [Build a custom module](https://docs.solspace.com/craft/freeform/v4/guides/building-a-custom-module/) that sends an email notification to a recipient based upon the selection of a Select/Checkbox Group/Radio fields' option using the `EVENT_AFTER_SUBMIT` event. We've created [a guide](https://docs.solspace.com/craft/freeform/v4/guides/building-a-custom-module/) that will get you 99% of the way there.

It's worth noting that Freeform 5 (entering beta in the coming weeks) will resolve this issue entirely. 🙂
